pages/common/rclone.md

```diff
@@ -3,27 +3,27 @@
 > Copy, synchronize or move files and directories to and from many cloud services.
 > More information: <https://rclone.org>.
 
+- Launch an interactive menu to setup rclone:
+
+`rclone config`
+
 - List contents of a directory on an rclone remote:
 
 `rclone lsf {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from local source to remote destination:
+- Copy file or directory from the local machine to remote destination:
 
-`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/destination_directory}}`
+`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from remote source to local destination:
+- Copy files changed within the past 24 hours to a remote to the local machine, asking user to confirm each file:
```

Comment:
```suggestion
- Copy files changed within the past 24 hours to a remote from the local machine, asking the user to confirm each file:
```
Is the copying happening from a remote to the local machine, or from a local machine to the remote?
Feel free to reword my suggestion for the same.